CMakeLists.txt

```diff
@@ -158,10 +158,12 @@ if (error EQUAL 0)
   ADD_SUBDIRECTORY(src/lib/orionTypes)
   ADD_SUBDIRECTORY(src/lib/ngsi)
   ADD_SUBDIRECTORY(src/lib/serviceRoutines)
+  ADD_SUBDIRECTORY(src/lib/serviceRoutinesV2)
```

Comment:
I just found a really good reason to not change the name of /lib/serviceRoutines. Let's see what you think ...

The name is used in hundreds of places, when including headers from the directory:

      #include "serviceRoutines/xxx.h"